Lib/defcon/objects/font.py

```diff
@@ -1382,11 +1408,92 @@ def _convertToFormatVersion1RoboFabData(self, libCopy):
                         finalValues.append([])
                     finalValues[-1].append(value)
                 hintData[libKey] = finalValues
-        for key, value in hintData.items():
+        for key, value in list(hintData.items()):
             if value is None:
                 del hintData[key]
         libCopy["org.robofab.postScriptHintData"] = hintData
 
+    # -----------------------------
+    # Serialization/Deserialization
+    # -----------------------------
+
+    def getDataForSerialization(self, **kwargs):
+        from functools import partial
+
+        simple_get = partial(getattr, self)
+        serialize = lambda item: item.getDataForSerialization()
+        serialized_get = lambda key: serialize(simple_get(key))
+
+        getters = (
```

Comment:
I'm cool with merging this part as is. I don't want to break anything just for hypothetical future situations. We can open an issue and improve it later.